resources/views/blade/form-row.blade.php

```diff
@@ -0,0 +1,90 @@
+<!-- form-row blade component -->
+@props([
+    'errors',
+    'label',
+    'name' => null,
+    'help_text' => null,
+    'label_style' => null,
+    'label_class' => 'col-md-3 col-sm-12 col-xs-12',
+    'div_style' => null,
+    'input_div_class' => 'col-md-8 col-sm-12',
+    'input_style' => null,
+    'item' => null,
+    'type' => 'text',
+    'placeholder' => null,
+    'maxlength' => 191,
+    'minlength' => null,
+    'min' => null,
+    'max' => null,
+    'step' => null,
+    'disabled' => false,
+    'error_offset_class' => 'col-md-7 col-md-offset-3',
+    'info_tooltip_text' => null,
+])
+
+<div {{ $attributes->merge(['class' => 'form-group']) }}>
+
+    <x-form-label
+            :$label
+            :for="$name"
+            :style="$label_style ?? null"
+            class="{{ $label_class }}"
+    />
+
+    <div {{ $attributes->merge(['class' => $input_div_class]) }} {{ ($div_style) ? $attributes->merge(['style' => $div_style]):'' }}>
```

Comment:
This can seemingly be simplified to improve readability

```suggestion
    <div {{ $attributes->merge(['class' => $input_div_class, 'style' => $div_style]) }} >
```